specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/DTL.json

```diff
@@ -2830,73 +2773,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DevTestLab/labs/{labName}/servicerunners": {
```

Comment:
Removal of existing APIs need ARM review.

Reply:
This was never intended to be published. No customers should be using this.

Reply:
This is problematic on two fronts:
1) Removing an API path is a breaking change. If it should not have been published, we will to confirm that there is no _actual_ usage from clients. 
2) GET of serviceRunner (by name) and PUT/DELETE are still defined. A List API is required for these. Removing this means the serviceRunner child resource is not contract compliant.

Reply:
This is a special kind of resource. It is created in order to leverage the MSI created along. It is PUT only from our own code. However, because ARM needs to be involved (ARM triggers the creation and deletion of the MSI), PUT/DELETE/GET needs to be public. However, we have code in our RP ensuring the CREATE call is actually coming from us. If this is really a concern, I can add the LIST back. But no one but our own RP is using it (that operation has been removed months back already).

Reply:
@joerage - are the GET calls made by external clients? Does this need to be documented in swagger at all if it is for service use only?

Reply:
Good point. I will remove this from swagger doc.